src/app/create-task/create-task.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+import {FormBuilder, FormGroup, ReactiveFormsModule, Validators} from "@angular/forms";
+import {Task} from "../task";
+import {NgClass, NgIf} from "@angular/common";
+
+@Component({
+  selector: 'app-create-task',
+  standalone: true,
+  imports: [
+    ReactiveFormsModule,
+    NgIf,
+    NgClass
+  ],
+  templateUrl: './create-task.component.html',
+  styleUrl: './create-task.component.scss'
+})
+export class CreateTaskComponent {
+  currentTask: FormGroup = this.fb.group({
```

Comment:
In angular usually structure your variables in this order:

Inputs,
Outputs,
Observables/BehaviorSubjects
rest of the variables